Atomic/util.py

```diff
@@ -241,9 +241,9 @@ def skopeo_layers(image, args=[], layers=[]):
             raise ValueError("Unable to interact with this registry: {}".format(r.stderr))
     except OSError:
         raise ValueError("skopeo must be installed to perform remote inspections")
-    except Exception as e:
+    finally:
```

Comment:
this changes the logic of the function and rm the directory even on success.  We should rm the temporary directory only on an exception.  I'll post another patch to address it.